doc/building.md

```diff
@@ -1,4 +1,60 @@
-Building nyan
-=============
+# Building nyan
+
+This project requires a C++14 compiler (e.g gcc >= 5.1 or clang >= 3.4) and uses Flex, make, automake, autoconf and libtool (any "recent" version should do).
+
+## Dependencies install
+
+###### Ubuntu 15.10 
+
+Required packages :
+```
+sudo apt-get update
+sudo apt-get install make libtool autoconf automake flex bison git
+```
+
+For gcc : `sudo apt-get install gcc-5 g++-5`
+
+For clang : `sudo apt-get install clang-3.5 clang++-3.5`
```

Comment:
Debian packaging........
Looks like you are right and we need to specify the clang version then, and just use gcc without a version. Thanks for the investigations.